src/core/util/log.cc

```diff
@@ -41,6 +41,7 @@
 #include <boost/log/sources/severity_channel_logger.hpp>
 #include <boost/log/support/date_time.hpp>
 #include <boost/make_shared.hpp>
+#include <boost/phoenix.hpp>
```

Comment:
As discussed in IRC, `boost::phoenix::bind` is a "lazy" evaluator so it apparently works well with boost expressions. We should aim for a `std::bind` or lambda replacement + refactor though (if possible) in order to reduce build overhead.

Reply:
I changed <boost/phoenix.hpp> to <boost/phoenix/bind.hpp>. Does that reduce build overhead?

(Asked that on IRC too.)